nixos/modules/services/admin/oddjobd.nix

```
@@ -0,0 +1,29 @@
+{ config, pkgs, lib, ... }:
+
+
+let
+  cfg = config.services.oddjobd;
+in
+{
+  options.services.oddjobd = {
```

```suggestion
  options.programs.oddjobd = {
```
this looks more like a program if it just needs some dbus setting